docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -327,6 +370,14 @@ You can also select the container runtime to use with:
 ./mvnw package -Pnative -Dquarkus.native.container-runtime=podman
 ----
 
+[TIP]
+====
+Building with Mandrel requires a custom builder image parameter to be passed additionally:
+[source,shell]
+----
+./mvnw package -Pnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-mandrel:20.1.0.1.Alpha2-java11
```

Comment:
I don't think we want that hardcoded here. It should be in the build-parent with the GraalVM version.

Reply:
Yeah, I wasn't happy about the hardcoding either. Even if it's in the build-parent, it'd be a hardcode of some sort. 

I wonder if we should have a a `20.1` quay tag that just links to the latest 20.1 release. That'd make it more palatable. @Karm @zakkak?

Reply:
+1

Reply:
Created https://github.com/graalvm/mandrel/issues/95